pom.xml

```diff
@@ -1353,13 +1353,13 @@
     <!-- Configure database config file to use. Could be
      h2, postgres, sqlserver, mysql, oracle, db2, postgres-postgis, jndi-postgres-postgis
      -->
-    <db.type>h2</db.type>
+    <db.type>postgres</db.type>
```

Comment:
extra commit ?

Reply:
Indeed. Let me remove it and force push.

Reply:
@cmangeat I've pushed the change.